src/adb/command/host/disconnect.coffee

```diff
@@ -5,7 +5,8 @@ class DisconnectCommand extends Command
   # Possible replies:
   # "No such device 192.168.2.2:5555"
   # ""
-  RE_OK = /^$/
+  # disconnected host:port
+  RE_OK = /^disconnected |(^$)/
```

Comment:
Could you get rid of the unnececessary `()`? I'd like to keep regexps as simple as possible.